web/wp-content/plugins/tmbi-adobe-dtm/js/adobe_dtm.js

```diff
@@ -0,0 +1,388 @@
+/**
+ * Created by jeysaravana on 2017-01-11.
+ */
+
+var qs = (function() {
+	query_string_url = window.location.search.substr(1).split('&');
+	if ( query_string_url === "" ) return {};
+	var query_strings = {};
+	for ( var i = 0; i < query_string_url.length; ++i ) {
+		var parameter = query_string_url[i].split('=', 2);
+		if ( parameter.length === 1 ) {
+			query_strings[parameter[0]] = "";
+		} else {
+			query_strings[parameter[0]] = decodeURIComponent( parameter[1].replace( /\+/g, " " ) );
+		}
+	}
+	return query_strings;
+})();
+
+//Nelio A/B Testing
+if ( qs['nabt'] !== undefined ) {
+	//TODO setup adobe variables.
+	//for original => nabt = 0
+	//for alternatice => nabt = 1
+	var test_name = 'tohdesign:a_newdesign';
+	if ( qs['nabt'] === '1' ) {
+		test_name = 'tohdesign:b_currentdesign';
+	}
+	digitalData.ab = digitalData.ab||{};
+	digitalData.ab.testing = test_name; //qs['nabt']
+}
+
+jQuery( document ).ready(function($) {
+
+	if ( qs['ehid'] !== undefined ) {
+		localStorage.newsletter 	= qs['ehid'];
+		digitalData.newsletter      = digitalData.newsletter||{};
+		digitalData.newsletter.ehid = qs['ehid'];
+	}
+	if ( typeof( localStorage.newsletter ) != 'undefined' && typeof qs['ehid'] == 'undefined' ){
+		digitalData.newsletter      = digitalData.newsletter||{};
+		digitalData.newsletter.ehid = localStorage.newsletter;
+	}
+
+	//RD Related changes
+	$("#more-from-category").find('a').each(function() {
```

Comment:
Why is this needed via JS? This will break when the theme changes to a different markup structure. Since it depends on specific markup selectors, it's probably better to have it in the theme layer (where the markup is created and controlled).